src/electron/listeners.js

```diff
@@ -216,14 +216,21 @@ const genValidatorKeysAndEncrypt = async (
         withdrawalAddr2StakeInfo[info.withdrawalSafeAddress].push(info)
     }
 
+    for (const [withdrawalSafeAddress, infoArray] of Object.entries(withdrawalAddr2StakeInfo)) {
+        const ids = infoArray.map((i) => i.validatorID)
+        validatorIDs.push(...ids)
+    }
+    const isUnique = storage.checkUniqueValidatorIndices(validatorIDs, address)
```

Comment:
these are `validator ID` which is internal representation of validator within ether.fi

`validator index` is a reserved term to refer their representation in beacon layer.